src/Microsoft.AspNetCore.Proxy/ProxyMiddleware.cs

```diff
@@ -1,22 +1,28 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using Microsoft.AspNetCore.Builder;
```

Comment:
No. It's my bad. My apologies. I've brainlessly used VS 2017 autosort refactoring.